app/models/manageiq/providers/embedded_ansible/automation_manager/configuration_script_source.rb

```diff
@@ -1,11 +1,122 @@
 class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptSource
+  FRIENDLY_NAME = "Ansible Automation Inside Project".freeze
+  REPO_DIR      = Rails.root.join("tmp", "git_repos")
```

Comment:
I think we probably need to work out how much data we're really going to be storing and based on that, where it makes sense to put this.

Additionally I would like to keep this contained enough that people can run it locally so I don't want to hard-code this to a location outside of the source tree.

Reply:
Agreed.

Of note, this was one of my earlier comments that was mostly asking questions about what was done, and getting some discussion going.

Reply:
Yeah I'm going to change it because git_worktree already has a location, but I think it's got "automate" in the name 😕 .  But, even so, I agree with @carbonin.

I think the more important thing here is that vmdb/tmp is only so big.  Since we are keeping these around we may need a dedicated disk.  The current usage of git_worktree gets around this because the only thing using it is automation domain import, and it only uses the disk space for the life of the import.

Reply:
So I think we should move this out of `tmp` as it's not really temporary, but the `/var` filesystem should be fine. [We have something like 12G of space there](https://github.com/ManageIQ/manageiq-appliance-build/blob/cbf9e707ea54c64d5a442a7093ebd2f547a78ac9/kickstarts/partials/main/disk_layout.ks.erb#L11) And that is where we were previously assuming tower would store it's data.

Reply:
Might want to also consider making this dynamic/settings back, because we probably don't want this always in `/var` in dev mode.

Anyway, resolved this since "conversation happend" :wink: